tests/integrations/test_dbm.py

```diff
@@ -113,3 +118,169 @@ def test_trace_payload_full(self):
                 self._assert_span_is_untagged(span)
             else:
                 self._assert_span_is_tagged(span)
+
+
+# @features.datastreams_monitoring_support_for_kafka
+# @scenarios.integrations
+class _Test_Dbm_Comment:
+    """ Verify DBM comment for given integration """
+
+    integration = None
+    operation = None
+    operation_batch = None
+    execute_batch = False
+
+    # comment generic info
+    dde = "system-tests"  # DD_ENV
+    ddps = "weblog"  # DD_SERVICE
+    ddpv = "1.0.0"  # DD_VERSION
+
+    def setup_dbm_comment(self):
+        self.r = weblog.get("/stub_dbm", params={"integration": self.integration, "operation": self.operation})
+        if self.r.text not in [None, ""]:
```

Comment:
Could you move this logic inside the test method, it'll be easier to use/debug/modify with the `replay` mode.